### DISCUSSION TITLE
ssrs reporting

### DISCUSSION THREAD

**Discussion by zprofreelancer** [serenity-is/Serenity#5387] [Help]
Can we integrate SSRS with serenity mvc?

Thanks.

**Comment by minhhungit** [comment]
Serenity is just a asp.net mvc framework (before), so whatever asp.net mvc framework can do, Serenity MVC can do
If you are using serenity MVC version then yes, you can integrate it with SSRS

But keep in mind that right now Serenity does not support MVC version anymore, it just support .Net Core version

**Comment by minhhungit** [comment]
You can search keyword "ssrs" on this repository, some people asked same question before, by reading old discusstion you might get the answer

![image](https://user-images.githubusercontent.com/2279508/105629732-55e32200-5e77-11eb-9805-a3ada83480d7.png)
